File: app/src/test/java/dictionary/CopyrightPageTest.java
package dictionary;

import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
import static org.junit.jupiter.api.Assertions.assertEquals;

import java.io.File;
import java.nio.file.Path;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.io.TempDir;

public class CopyrightPageTest {

    @Test
    public void createPage(@TempDir Path tempDir){
        assertDoesNotThrow(() -> new CopyrightPage(tempDir.toString() + File.separator).build());
        File[] outputFiles = tempDir.toFile().listFiles();
        assertEquals(1, outputFiles.length);
        assertEquals(CopyrightPage.FILE_NAME, outputFiles[0].getName());
    }
    
}

File: app/src/test/java/dictionary/CopyrightPageTest.kt
package dictionary

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.io.TempDir
import java.io.File
import java.nio.file.Path

class CopyrightPageTest {
    @Test
    fun createPage(@TempDir tempDir: Path) {
        Assertions.assertDoesNotThrow { CopyrightPage(tempDir.toString() + File.separator).build() }
        val outputFiles = tempDir.toFile().listFiles()
        Assertions.assertNotNull(outputFiles)
        Assertions.assertEquals(1, outputFiles!!.size)
        Assertions.assertEquals(CopyrightPage.FILE_NAME, outputFiles[0].name)
    }
}